ElectronClient/app/app.js

```diff
@@ -551,6 +571,13 @@ class Application extends BaseApplication {
 			shim.isMac() ? noItem : newNotebookItem, {
 				type: 'separator',
 				visible: shim.isMac() ? false : true
+			}, {
+				label: _('Templates'),
+				visible: shim.isMac() ? false : true,
```

Comment:
Any reason why this is not visible on macOS?

Reply:
I was just following the surrounding code, I add the templates in `rootMenuFileMacOs` for Mac os just below this code

Reply:
This is fine then, I missed that in the context of the diff.